content/en/continuous_integration/setup_tests/dotnet.md

```diff
@@ -72,6 +72,57 @@ The following list shows the default values for key configuration settings:
 
 All other [Datadog Tracer configuration][2] options can also be used.
 
+### Collecting Git metadata
+
+Git information is required for the proper visualization of test results, and used to group them by repository, branch and commit. This is automatically collected by the test instrumentation using CI provider environment variables and the local `.git` folder at the project path, if available.
+
+If running tests in non-supported CI providers or with no `.git` folder, Git information can be set manually using environment variables. These environment variables take precedence over any autodetected information.
+The supported environment variables for providing Git information are the following:
+
+`DD_GIT_REPOSITORY_URL`
+: URL of the repository where the code is stored. Both HTTP and SSH URLs are supported.
```

Comment:
I just realized the `<br/>` tags are missing so the example appears inline instead of in a new line.

```suggestion
: URL of the repository where the code is stored. Both HTTP and SSH URLs are supported.<br/>
```